app/javascript/elm/tests/SessionsListTests.elm

```diff
@@ -0,0 +1,201 @@
+sessionWithTitle title =
+    { session | title = title }
+
+
+viewTests : Test
+viewTests =
+    describe "view"
+        [ fuzz2 string string "session titles are displayed in the list" <|
+            \title1 title2 ->
+                { sessions = [ sessionWithTitle title1, sessionWithTitle title2 ], selectedSessionId = Nothing }
+                    |> view
+                    |> Query.fromHtml
+                    |> Query.contains
+                        [ Html.text title1
+                        , Html.text title2
+                        ]
+        , fuzz int "selected session is selected in the list" <|
+            \id ->
+                { sessions = [ sessionWithId id, sessionWithId (id + 1) ], selectedSessionId = Just (id + 1) }
+                    |> view
+                    |> Query.fromHtml
+                    |> Query.findAll [ Slc.all [ Slc.tag "input", Slc.checked True ] ]
+                    |> Query.count (Expect.equal 1)
+        , fuzz (intRange 1 10) "with modulo 50 sessions in the model the load more button is shown" <|
+            \times ->
+                { sessions = List.repeat (50 * times) session, selectedSessionId = Nothing }
+                    |> view
+                    |> Query.fromHtml
+                    |> Query.contains [ Html.text "Load More..." ]
+        , test "with 0 sessions in the model the load more button is not shown" <|
+            \times ->
+                { sessions = [], selectedSessionId = Nothing }
+                    |> view
+                    |> Query.fromHtml
+                    |> Query.findAll [ Slc.text "Load More..." ]
+                    |> Query.count (Expect.equal 0)
+        ]
+
+
+updateTests : Test
+updateTests =
+    describe "update"
+        [ fuzz int "with no selections ToggleSessionSelection selects the session" <|
+            \id ->
+                let
+                    model =
+                        { sessions = [ sessionWithId id, sessionWithId (id + 1) ], selectedSessionId = Nothing }
+
+                    expected =
+                        { model | selectedSessionId = Just (id + 1) }
+                in
+                model
+                    |> update (ToggleSessionSelection (id + 1))
+                    |> Tuple.first
+                    |> Expect.equal expected
+        , fuzz int "when session was selected ToggleSessionSelection deselects it" <|
+            \id ->
+                let
+                    model =
+                        { sessions = [ sessionWithId id, sessionWithId (id + 1) ], selectedSessionId = Just (id + 1) }
+
+                    expected =
+                        { model | selectedSessionId = Nothing }
+                in
+                model
+                    |> update (ToggleSessionSelection (id + 1))
+                    |> Tuple.first
+                    |> Expect.equal expected
+        , fuzz int "when another session was selected ToggleSessionSelection selects the new one" <|
```

Comment:
I don't see that happening, I'm seeing a "You can't select multiple sessions" message

Reply:
I guess because it wasn't changed on the js side?

Reply:
perhaps it would be nice to just get rid of that part of code from js, the new behaviour makes much more sense

Reply:
Were you running that on your machine? If so I guess you have the old partials cached

Reply:
that's right.

Reply:
Just so you know, until we move out of what we have we always need to do what's documented here https://github.com/HabitatMap/AirCasting#assets-version-bumping